input/resources/StructureDefinition-extension-qualification.json

```diff
@@ -0,0 +1,232 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "Qualification",
+  "url": "http://hl7.org/fhir/us/ndh/StructureDefinition/base-ext-qualification",
```

Comment:
change to "url": "http://signalbhn.org/fhir/StructureDefinition/organization-qualification"

Reply:
Changed